### DISCUSSION TITLE
Fermata auto 1.8.11 and AA 7.3.620524

### DISCUSSION THREAD

**Discussion by mduilio** [AndreyPavlenko/Fermata#152] [Q&A]
Hi Pavlenko!
Is the latest version of Fermata working with the latest version of AA? I will only upgrade after making sure it works. Thank you!

**Comment by LasGeo64** [comment]
The version of AA & fermata is the same as mime.
And I just find it can work well this morning.
[Step for reference]
Old AA installed + fermata 1.8.11 using AAAD (not work)
Android force to upgrade 
Then AA 7.3.620524 + fermata1.8.11 can work.
Really surprised . But it is just my experience for reference.
